statusHandler/persister/common.go

```diff
@@ -3,10 +3,21 @@ package persister
 // GetUint64 will try to convert an interface type in a uint64
 // in case of failure wil return 0
 func GetUint64(data interface{}) uint64 {
-	value, ok := data.(float64)
+	value, ok := data.(uint64)
 	if !ok {
 		return 0
 	}
 
-	return uint64(value)
+	return value
+}
+
+// GetString will try to convert an interface type in a string
+// in case of failure wil return 0
+func GetString(data interface{}) string {
+	value, ok := data.(string)
+	if !ok {
+		return "0"
```

Comment:
It was used for passing the number to a big int. Now I extracted a method for converting a string to big Int but I will leave this method as it can be useful in the future. changed to `""`